layouts/default.vue

```diff
@@ -1,5 +1,13 @@
 <template>
   <div class="bg-ink-400 flex flex-col min-h-screen antialiased text-vanilla-300">
+    <banner cta-link="https://deepsource.io/discover">
+      <span slot="text"
+        >Find issues in any open source repository and fix them automatically with Discover.</span
+      >
+      <template slot="cta"
+        >LEARN MORE <ExternalLinkIcon class="h-4 md:h-6 ml-1 md:ml-2 text-vanilla-400"
```

Comment:
Margins and positioning should be handled in the <banner> component, and not the slot.

Reply:
I am removing the margins that are unnecessary but all of them cannot be shifted inside the Banner component as these are specific to the link, `LEARN MORE` and it's icon.